src/app/recruitment/[id]/components/JobDetailDate.tsx

```
@@ -12,10 +18,12 @@ const JobDetailDate = () => {
             height={20}
             alt="calender"
           />
-          0월 00일 00:00 마감
+          {expiredDate === '무관' || '상시채용'
+            ? expiredDate
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)